packages/curve-ui-kit/src/features/user-profile/settings/advanced-mode/AdvancedMode.tsx

```diff
@@ -0,0 +1,17 @@
+import { t } from '@lingui/macro'
+import Switch from '@mui/material/Switch'
+
+import { Label, Setting } from '../Setting'
+import useUserProfileStore from '../../store'
+
+export const AdvancedMode = () => {
+  const isAdvancedMode = useUserProfileStore((state) => state.isAdvancedMode)
+  const setAdvancedMode = useUserProfileStore((state) => state.setAdvancedMode)
+
+  return (
+    <Setting>
+      <Label>{t`Advanced Mode`}</Label>
+      <Switch checked={isAdvancedMode} onChange={() => setAdvancedMode(!isAdvancedMode)} color="primary" size="small" />
```

Comment:
I discovered that the correct way to do this is using the `FormControlLabel`  ([like here](https://github.com/curvefi/curve-frontend/blob/e400655e2ac3c472171b0a7ae4e1f98964302736/packages/curve-ui-kit/src/features/switch-advanced-mode/AdvancedModeSwitcher.tsx#L12))

```suggestion
      <FormControlLabel control={<Switch checked={isAdvancedMode} onChange={() => setAdvancedMode(!isAdvancedMode)} color="primary" size="small" />} label={t`Advanced Mode`}>
```

Reply:
![image](https://github.com/user-attachments/assets/7ec88f90-56ec-4770-a4b7-78e8e1bd94b3)

Not sure it works neatly with the grid I've just set up. If you insist on using it together with a proper form I'd rather do that in a diff PR as I'm sure other things will need to be changed as well.